{"instance_id": "UDMIS043", "instance_filename": "UDMIS043_25_N25_W1.0_R0.4.qubo.json", "solver_name": "QUBOIBMSolver", "args": ["{'maxiter': 50}", "BOBYQA", "0", "0", "0", "0", "3", "1000", "1000", "ibmq_qasm_simulator", "ibm_quantum", "None", "8h", "False", "1", "1"], "timestamps": {"start": 1699982690.582213, "end": 1699983235.969709}, "solver": {"exception": "'Failed to run program: \"HTTPSConnectionPool(host=\\'api.quantum-computing.ibm.com\\', port=443): Max retries exceeded with url: /runtime/jobs (Caused by ResponseError(\\'too many 524 error responses\\'))\"'", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench.py\", line 152, in run\n    solver = self._run(instance_id, *args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 204, in _run\n    raise e\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 200, in _run\n    res = self._run_qpu_estimator(session, estimator_options, optimizer_name, optimizer_kwargs,\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 113, in _run_qpu_estimator\n    res = optimizer_obj.minimize(fun=lambda params: cost_func(params, ansatz, qubitOp, estimator), x0=x0,\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\qiskit\\algorithms\\optimizers\\bobyqa.py\", line 72, in minimize\n    res, history = skq.minimize(\n                   ^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\skquant\\opt\\__init__.py\", line 56, in minimize\n    return optimizer.minimize(func, x0, bounds, budget, options, **optkwds)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\skquant\\opt\\_pybobyqa.py\", line 33, in minimize\n    result = pybobyqa.solve(\n             ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\pybobyqa\\solver.py\", line 811, in solve\n    solve_main(objfun, x0, args, xl, xu, npt, rhobeg, rhoend, maxfun, nruns, nf, nx, nsamples, params,\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\pybobyqa\\solver.py\", line 107, in solve_main\n    f0 = eval_objective(objfun, remove_scaling(x0, scaling_changes), args, eval_num=nf, pt_num=nx,\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\pybobyqa\\util.py\", line 53, in eval_objective\n    f = objfun(x, *args)\n        ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\SQCommon\\_objective.py\", line 24, in __call__\n    result = self.objective(par)\n             ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 113, in <lambda>\n    res = optimizer_obj.minimize(fun=lambda params: cost_func(params, ansatz, qubitOp, estimator), x0=x0,\n                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\Documents\\qopt\\benchmark\\bench_qubo.py\", line 155, in cost_func\n    estimator_result = estimator.run(ansatz, hamiltonian, parameter_values=params).result()\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\estimator.py\", line 147, in run\n    return super().run(\n           ^^^^^^^^^^^^\n  File \"C:\\Users\\admin\\AppData\\Roaming\\Python\\Python311\\site-packages\\qiskit\\primitives\\base\\base_estimator.py\", line 188, in run\n    return self._run(\n           ^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\estimator.py\", line 185, in _run\n    return self._run_primitive(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\base_primitive.py\", line 160, in _run_primitive\n    return self._session.run(\n           ^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\session.py\", line 36, in _wrapper\n    return func(self, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\session.py\", line 161, in run\n    job = self._service.run(\n          ^^^^^^^^^^^^^^^^^^\n  File \"C:\\ProgramData\\anaconda3\\envs\\qopt\\Lib\\site-packages\\qiskit_ibm_runtime\\qiskit_runtime_service.py\", line 1028, in run\n    raise IBMRuntimeError(f\"Failed to run program: {ex}\") from None\nqiskit_ibm_runtime.exceptions.IBMRuntimeError: 'Failed to run program: \"HTTPSConnectionPool(host=\\'api.quantum-computing.ibm.com\\', port=443): Max retries exceeded with url: /runtime/jobs (Caused by ResponseError(\\'too many 524 error responses\\'))\"'\n", "fail_timestamp": 1699983235.969709}, "success": false, "version": "2.0.1"}